examples/truffle/yarn.lock

```diff
@@ -3,25 +3,25 @@
 
 
 "@babel/helper-module-imports@^7.12.1":
-  version "7.12.5"
-  resolved "https://registry.yarnpkg.com/@babel/helper-module-imports/-/helper-module-imports-7.12.5.tgz#1bfc0229f794988f76ed0a4d4e90860850b54dfb"
-  integrity sha512-SR713Ogqg6++uexFRORf/+nPXMmWIn80TALu0uaFb+iQIUoR7bOC7zBWyzBs5b3tBBJXuyD0cRu1F15GyzjOWA==
+  version "7.12.13"
+  resolved "https://registry.npmjs.org/@babel/helper-module-imports/-/helper-module-imports-7.12.13.tgz"
```

Comment:
Weird, it looks like all `s/yarnpkg.com/npmjs.org/` for the resolved URLs for the packages. What version of Yarn are you using?

Reply:
I am on `1.22.10`. I redid the upgrade this time without using `yarn upgrade` which has fixed this.